validator/rpc/wallet.go

```diff
@@ -126,7 +126,7 @@ func (s *Server) WalletConfig(_ context.Context, _ *empty.Empty) (*pb.WalletResp
 	switch s.wallet.KeymanagerKind() {
 	case keymanager.Derived:
 		keymanagerKind = pb.KeymanagerKind_DERIVED
-	case keymanager.Imported:
+	case keymanager.Local:
```

Comment:
we should add a note or something about the api not supporting this type so we don't break the ui on
https://github.com/prysmaticlabs/prysm/issues/9883, it can be addressed together on another pr

Reply:
It won't break the UI because the actual response will contain pb.KeymanagerKind_IMPORTED. We didn't change the UI-related types so it should be safe